src/Utilities/hooks/useTableTools/useTableTools.js

```diff
@@ -42,11 +42,8 @@ const useTableTools = (
   );
 
   const { tableProps: actionResolverTableProps } = useActionResolverWithItems({
-    //items: filteredAndSortedItems(identifiedItems, filter, sorter),
     items: filteredAndSortedItems(items, filter, sorter),
     ...options,
-    //...tablePropsOption,
-    ...tableProps,
```

Comment:
Is this intentional? Why are we removing tableProps?

Reply:
Because, I have found out that `options` argument already includes both `tableProps` and `actionResolver` variables that are actually used in `useActionResolverWithItems` hook. Thus, we do not have to pass here `tableProps` again and again.

![Screenshot 2024-07-19 at 9 51 51](https://github.com/user-attachments/assets/25164bbd-213e-41d1-a1d5-535ae7fe3105)